e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportPage.js

```diff
@@ -63,6 +63,10 @@ const utilisationReportPage = {
       paymentCurrencyRadioInput: (currencyCode) => cy.get(`input[type="radio"][data-cy="currency-${currencyCode}"]`),
       paymentCurrencyError: () => cy.get('[data-cy="payment-details--filter-payment-currency-error"]'),
       noRecordsMatchingFiltersText: () => cy.get('[data-cy="payment-details-no-records-matching-filters-text"]'),
+      actionBar: () => cy.get('[data-cy="payment-details--filters-action-bar"]'),
+      actionBarItem: (value) => cy.get(`[data-cy="payment-details--filters-action-bar"] a:contains(${value})`),
+      clearFiltersLink: () => cy.get('a:contains("Clear filters")'),
```

Comment:
Can we use a `data-cy` attribute instead of `a:contains`?

Reply:
The nunjucks documentation isn't great for MOJ components - can we try adding a `data-cy` to the nunjucks - if it doesn't work then happy with the current selector

```
clearLink: {
  text: "Clear filters",
  attributes: {
    "data-cy": "testing"
  }
}
```